src/main/kotlin/com/terraformation/backend/documentproducer/db/variable/VariableImporter.kt

```diff
@@ -366,6 +366,22 @@ class VariableImporter(
                 .execute()
           }
         }
+
+        // If there's no deliverable ID on this sheet, and this variable replaces an existing one,
+        // then the variable's deliverable(s) might have been specified on the deliverables sheet.
+        // The new variable should replace the old one in deliverables' variable lists.
+        //
+        // TODO: Remove the deliverableId null check here when we stop supporting deliverable IDs
```

Comment:
Yes! Was going to point out that there may be a chicken and egg problem here. Variables sheet depending on deliverables and vice versa

Reply:
It's a little messy if we do a gradual transition but the end state will be that the dependency direction will be flipped and you'll need to import variables before deliverables.